gradle-jdks-setup/src/integrationTest/resources/testing-script.sh

```diff
@@ -3,7 +3,18 @@
 set -e
 
 /root/.gradle/gradle-jdks/amazon-corretto-11.0.21.9.1/bin/java -version
-if [ -f /palantir.crt ];then
-  echo "Palantir cert:" $(/root/.gradle/gradle-jdks/amazon-corretto-11.0.21.9.1/bin/keytool -keystore /root/.gradle/gradle-jdks/amazon-corretto-11.0.21.9.1/lib/security/cacerts -list -alias gradlejdks_palantir3rd-generationrootca -storepass changeit)
+# Running again the gradle-jdk-setup to check the JAVA_HOME.
+
+# >>> Gradle JDK setup >>>
+# !! Contents within this block are managed by 'palantir/gradle-jdks' !!
+if [ -f gradle/gradle-jdks-setup.sh ]; then
+    if ! . gradle/gradle-jdks-setup.sh; then
+        echo "Failed to set up JDK, running gradle/gradle-jdks-setup.sh failed with non-zero exit code"
+        exit 1
+    fi
+    # Setting JAVA_HOME to the gradle daemon to make sure gradlew uses this jdk for `JAVACMD`
+    JAVA_HOME="$GRADLE_DAEMON_JDK"
 fi
-echo "Example.com cert:" $(/root/.gradle/gradle-jdks/amazon-corretto-11.0.21.9.1/bin/keytool -keystore /root/.gradle/gradle-jdks/amazon-corretto-11.0.21.9.1/lib/security/cacerts -list -alias gradleJdks_example.com -storepass changeit)
+# <<< Gradle JDK setup <<<
```

Comment:
Is this manually placed here or is it kept up to date with `gradlew-patch.sh` somehow?